vocabulary-draft/introduction.md

```diff
@@ -0,0 +1,22 @@
+This document intends to describe a set of events related to different phases of a Continuous Delivery process. 
```

Comment:
NIT: what about a title here?

Reply:
I've proposed one title, but I am not 100% sure of it.. any suggestions? I was also thinking about Continuous Delivery Foundation Events Vocabulary